### ISSUE TITLE
Where to get weights?

### ISSUE THREAD

**Issue by fangxiao-dev** [anshkumar/yolact#5]
Hi, thanks for your work. But where could I got the weights file, e.g. `saved_model_0.19968511` ? I don't find link in your description. 

Thanks for your time!

**Comment by anshkumar** [comment]
Currently, I'm not providing pre-trained checkpoints. Will add coco pre-trained checkpoints in future.

**Comment by Ankur-singh** [comment]
any updates on it? At least can you tell how to train the model & generate the weights.

**Comment by anshkumar** [comment]
@Ankur-singh For training you have to just change the path in `train.sh` file. And for creating tfrecord you have to use `create_coco_tfrecord.sh` file.

**Comment by Ankur-singh** [comment]
I think, one will have to download the coco dataset and unzip it before creating tfrecords. Because, I cannot find the code to download the dataset. Please correct me if am wrong or if I am missing something.

**Comment by anshkumar** [comment]
Yes, you have to download the dataset.

**Comment by shashank-sharma333** [comment]
@Ankur-singh Were you able to train the model?
I am getting the below error while running train.sh with the required parameters

tensorflow.python.framework.errors_impl.InvalidArgumentError: buffer_size must be greater than zero.

**Comment by Ankur-singh** [comment]
@shashank-sharma333 yes I was able to train the model. I had to make quite a few changes to make it work. But in the process, I never came across this error. 😅

**Comment by shashank-sharma333** [comment]
Thank you @Ankur-singh for your comment. Which environment you are using for the training model? Also, Can you list down the changes you made? It would be helpful.

**Comment by Ankur-singh** [comment]
I have a 3060Ti, so I followed GPU installation instruction from the readme file. Its pretty straight forward. Just make sure your compile the protobufs. Just run this from the root directory of the project

```bash
protoc protos/*.proto --python_out=.
```

Regarding the changes, TBH, I don't remember all of them. It was a couple of weeks ago. One was regarding SGD optimizer, the API has changed, so you should use legacy version. And there were quite a few data type issue, you should be able to fix them with basic type casting. Hope this helps.
